### ISSUE TITLE
Docs snippet styling unexpected

### ISSUE THREAD

**Issue by skovy** [vercel-labs/ai#266]
When following various parts of the docs (eg: https://sdk.vercel.ai/docs/api-reference/use-chat), there is this snippet of code:

```tsx
      <form onSubmit={handleSubmit}>
        <label>
          Say something...
          <input
            className="fixed w-full max-w-md bottom-0 border border-gray-300 rounded mb-8 shadow-xl p-2"
            value={input}
            onChange={handleInputChange}
          />
        </label>
        <button type="submit">Send</button>
      </form>
```

It renders like this:

![image](https://github.com/vercel-labs/ai/assets/5247455/82c70078-5860-49ab-9634-47b66566b34d)

This caught me off guard and made me think I copied something wrong or Tailwind was misconfigured.

It looks like this change was made in #114

I was going to open a PR to update the styles with something like the following:

```tsx
      <form
        onSubmit={handleSubmit}
        className="fixed bottom-0 w-full max-w-md mb-8"
      >
        <label>Say something...</label>
        <div className="flex gap-2">
          <input
            className="w-full border border-gray-300 rounded shadow-xl p-2"
            value={input}
            onChange={handleInputChange}
          />
          <button type="submit" className="bg-black text-white p-2 rounded">Send</button>
        </div>
      </form>
```

I second-guessed this being the right solution (adding more complexity to the docs with more styles) and wanted to open an issue to see the desired solution. 

- Revert the previous PR and add `aria-label` attributes so if somebody does copy/paste there is at least some label for accessiblity 
- Update all the file snippets to the above (I noticed there are several snippets that weren't updated like in the README and docs for other frameworks like Vue).
- Something else?

I'd be happy to make the update with the desired solution!

**Comment by jiarenlim** [comment]
thank you for the code update, however i encounter one weird problem, i use the starter for the huggingface and nothing pop up on my screen after i ask anything and also no responses from the AI was shown or streamed which im just looking at a blank white screen. But I do check out there are responses from AI in the network  tab in dev tools, do you know why is that? 
![image](https://github.com/vercel-labs/ai/assets/67389513/aa015358-fdd4-4946-ad00-8b406306fb0a) 
![image](https://github.com/vercel-labs/ai/assets/67389513/de751dcb-a74a-455f-b0e4-78f8ca08a991)
![Screenshot 2023-07-03 132740](https://github.com/vercel-labs/ai/assets/67389513/82ecbadd-c6f0-41b1-8312-21b67ad43a39)

**Comment by MaxLeiter** [comment]
The second option seems best to me. A PR would be appreciated!

**Comment by skovy** [comment]
Looks like this was mostly fixed by removing the wrong styles in ff70b9746bc4f6fb99c2d126a0a9a5ec6988110c

**Comment by lgrammel** [comment]
## Identify issue type

Identified as **Documentation**  
Confidence: **High**

### Rationale
The issue concerns correcting code snippets and styling presented in the documentation, rather than changing AI SDK product behavior.

### Evidence
- The documented `useChat` form snippet renders with the label and button separated from the fixed input, making the copied result appear misconfigured.
- The reporter proposes correcting the styling across documentation snippets, including README and framework-specific docs.
- A maintainer agreed that updating all affected snippets was the preferred solution, and a later comment notes the incorrect styles were mostly removed.
